aspnet/mvc/views/layout.rst

```diff
@@ -0,0 +1,111 @@
+
+Sections
+^^^^^^^^
+
+A layout can optionally reference one or more *sections*, by calling ``RenderSection``. Sections provide a way to organize where certain page elements should be placed. Each call to ``RenderSection`` can specify whether that section is required or optional. Individual views specify the content to be rendered within a section using the ``@section`` Razor syntax. If a view defines a section, it must be rendered (or an error will occur).
+
+An example ``@section`` definition in a view:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/Manage/VerifyPhoneNumber.cshtml
+  :language: html
+  :lines: 28-30
+
+In the code above, validation scripts are added to the ``scripts`` section on a view that includes a form. Other views in the same application might not require any additional scripts, and so wouldn't need to define a scripts section.
+
+Sections only flow from views. They cannot be referenced from partials, view components, or other parts of the view system.
+
+.. _viewimports:
+
+Importing Shared Directives
+---------------------------
+
+Views can use Razor directives to do many things, such as specifying namespaces or performing :doc:`dependency injection <dependency-injection>`. Directives used by many views may be specified in a ``_ViewImports.cshtml`` file.  The ``_ViewImports`` file supports the following directives:
+
+- addTagHelper
+- removeTagHelper
+- tagHelperPrefix
+- using
+- model
+- inherits
+- inject
+
+The file does not support other Razor features, such as functions and section definitions.
+
+A sample ``_ViewImports.cshtml`` file:
+
+.. literalinclude:: ../../../common/samples/WebApplication1/src/WebApplication1/Views/_ViewImports.cshtml
+  :language: html
+
+The ``_ViewImports.cshtml`` file for an ASP.NET Core MVC app is typically placed in the ``Views`` folder root. A ``_ViewImports.cshtml`` file can be placed within a controller-associated view folder, in which case it will only be applied to views within that folder. ``_ViewImports`` files are run first at the root level, and then for a ``_ViewImports`` file specified in the controller-associated folder, so settings specified at the root level may be overridden at the folder level.
+
+For example, if a root level ``_ViewImports.cshtml`` file specifies ``@model`` and ``@addTagHelper``, and another ``_ViewImports.cshtml`` file in the controller-associated folder of the view specifies a different ``@model`` and adds another ``@addTagHelper``, the view will have access to both tag helpers and will use the latter ``@model``.
+
+If multiple ``_ViewImports.cshtml`` files are run for a view, combined behavior of the directives included in the ``ViewImports.cshtml`` files will be as follows:
+
+- addTagHelper and remoteTagHelper: all run, in order
+- tagHelperPrefix, model, inherits: the closest one to the view overrides any others
```

Comment:
makes it sound like `tagHelperPrefix` overrides `model`. Might just make sense to expand this line and the previous for the sake of clarity.